### PR TITLE
Feature/ss design changes

### PR DESCRIPTION
**Pull request by ens-ap5**
## Type

- [ ] Bug fix
- [ ] New feature
- [x] Improvement to existing feature

## Related JIRA Issue(s)

https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-254

## Importance

Needs merging immediately, for July release.

## Description

CSS tweaks to search panel and popular species icons

## Views affected

Species Selector page only

### Other effects

No test required - cosmetic changes only

- [ ] I have checked any requirements for Google Analytics
- [ ] I have created any required tests
- [ ] The PR has as its final commit a WASM/JS update commit if any Rust files were updated.

## Possible complications

None.

### CODE REVIEW

File: src/ensembl/src/content/app/species-selector/containers/species-search-panel/SpeciesSearchPanel.scss [hunk 1]
```diff
@@ -2,9 +2,10 @@
 
 .speciesSearchPanel {
   background-color: $ens-light-grey;
-  height: 270px;
+  height: 235px;
   padding: 65px 40px;
-  padding-left: 100px; // FIXME: should this be shared variable? it will be reused by PopularSpeciesPanel
+  padding-left: $global-padding-left;
+  box-shadow: 0 3px 5px $global-box-shadow;
```
Comment (azangru) [inline src/ensembl/src/content/app/species-selector/containers/species-search-panel/SpeciesSearchPanel.scss]: Should `$global-box-shadow` variable be also used in the Home screen scss [here](https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/content/home/Home.scss#L15) then?

**Comment by azangru** [comment]
image

The grid of 42 popular species is now not evenly split into three equal-sized rows

File: src/ensembl/src/content/app/species-selector/containers/species-search-panel/SpeciesSearchPanel.scss (hunk 1 above)
Comment (ens-ap5) [reply src/ensembl/src/content/app/species-selector/containers/species-search-panel/SpeciesSearchPanel.scss]: Yes, I think so - I can do a tidy-up for next release.

**Comment by ens-ap5** [comment]
Re the rows - doesn't it depend on your browser window size? If the design is responsive, we can't really control that kind of thing.

**Comment by azangru** [comment]
> Re the rows - doesn't it depend on your browser window size? If the design is responsive, we can't really control that kind of thing.

We can control the maximum width of this block, so that it's 3 rows of 14 buttons in the best-case scenario (on a sufficiently wide screen) — and whatever fits the screen width on smaller screens.

**Comment by ens-ap5** [comment]
Would that fall under the auspices of breakpoints? That's on my list of things to do for August (ENSWBSITES-205)

**Comment by azangru** [comment]
It might, but for the time being, can we tweak these values in [PopularSpeciesPanel.scss](https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/content/app/species-selector/containers/popular-species-panel/PopularSpeciesPanel.scss#L4-L8), so that its width is no greater than the combined width of 14 popular species buttons plus gaps between them?

It might even be worth importing the values of popular species button dimensions into the PopularSpeciesPanel.scss.

**Comment by ens-ap5** [comment]
Made additional changes as per conversation.

**Pull request merged**